src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleTypeSymbol.vb

```diff
@@ -664,21 +668,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
 
                                     Dim defaultName As String = TupleMemberName(tupleFieldIndex + 1)
-                                    ' Add a field with default name if the given name Is different
-                                    If namesOfVirtualFields(tupleFieldIndex) <> defaultName Then
-                                        ' The name given doesn't match default name ItemTupleTypeSymbol.RestPosition, etc.
-                                        members.Add(New TupleRenamedElementFieldSymbol(Me, FieldSymbol, defaultName, -members.Count - 1, Nothing))
-                                    End If
 
-                                    ' Add a field with the given name
-                                    Dim location = If(_elementLocations.IsDefault, Nothing, _elementLocations(tupleFieldIndex))
+                                    ' Add a field with default name regardless
+                                    members.Add(New TupleRenamedElementFieldSymbol(Me, FieldSymbol, defaultName, -members.Count - 1, Nothing))
```

Comment:
This seems strange. I though `TupleRenamedElementFieldSymbol` should only be used when a friendly name is specified.